README.md

```diff
@@ -64,21 +64,57 @@ head over to [our infrastructure documentation](/doc/how-to/manage-infrastructur
 
 ## E2E tests
 
-Install Playwright
+Install Playwright:
 
 ```bash
 npm install
 npx playwright install
 ```
 
-Test with and without UI
+### Running locally against the `dev` environment
+
+You will need to have the following **environment variables** in your local `.env` file:
+
+```
+HMPPS_AUTH_USERNAME=AP_USER_TEST_1
```

Comment:
I think it would be worth updating the `.env.example` file with these env vars as part of this PR